storm-client/src/jvm/org/apache/storm/utils/CuratorUtils.java

```diff
@@ -75,29 +72,7 @@ public List<ACL> getAclForPath(String s) {
 
     protected static void setupBuilder(CuratorFrameworkFactory.Builder builder, final String zkStr, Map<String, Object> conf,
                                        ZookeeperAuthInfo auth) {
-        List<String> exhibitorServers = ObjectReader.getStrings(conf.get(Config.STORM_EXHIBITOR_SERVERS));
-        if (!exhibitorServers.isEmpty()) {
-            // use exhibitor servers
-            builder.ensembleProvider(new ExhibitorEnsembleProvider(
-                new Exhibitors(exhibitorServers,
-                        ObjectReader.getInt(conf.get(Config.STORM_EXHIBITOR_PORT)),
-                        new Exhibitors.BackupConnectionStringProvider() {
-                            @Override
-                            public String getBackupConnectionString() throws Exception {
-                                // use zk servers as backup if they exist
-                                return zkStr;
-                            }
-                        }),
-                new DefaultExhibitorRestClient(),
-                ObjectReader.getString(conf.get(Config.STORM_EXHIBITOR_URIPATH)),
-                ObjectReader.getInt(conf.get(Config.STORM_EXHIBITOR_POLL)),
-                new StormBoundedExponentialBackoffRetry(
```

Comment:
I also see these in defaults.yaml.  Should they be removed?

storm.exhibitor.port: 8080
storm.exhibitor.poll.uripath: "/exhibitor/v1/cluster/list"

Reply:
fixed